src/styles/blocks/calender.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
```

Comment:
you need to center calendar both vertically and horizontally.
Currently vertically it's not centered.